modules/relax/ModuleConfig.cfc

```diff
@@ -155,9 +155,13 @@ component{
 			sessionsEnabled	= getApplicationMetadata().sessionManagement
 		};
 
-		// Apend it
+		// Append it
 		structAppend( configStruct.relax, relaxDSL, true );
 
+		// Set our server type for compatibility checks on java objects
```

Comment:
@lmajano : Can you clarify? This is a boilerplate way I've seen of handling this for a number of the core modules in the past, so I guess I just copied that as my usual method. If there's a better way, I'd be glad to implement it that way.